lib/omniauth/strategies/apple.rb

```diff
@@ -76,13 +76,13 @@ def id_info
       def fetch_jwks
         uri = URI.parse('https://appleid.apple.com/auth/keys')
         response = Net::HTTP.get_response(uri)
-        { keys: JSON.parse(response.body)['keys'].map { |key| deep_symbolize(key) } }
+        JSON.parse(response.body, symbolize_names: true)
       end
 
       def verify_nonce!(payload)
-        return unless payload[:nonce_supported]
+        return unless payload['nonce_supported']
 
-        return if payload[:nonce].present? && payload[:nonce] != stored_nonce
```

Comment:
@nhosoya @btalbot 

before
```
payload[:nonce] != stored_nonce
```

after
```
payload['nonce'] == stored_nonce
```

expect `payload nonce` and `stored_nonce` match? 
Is that the right change?